agent/api/task_unix.go

```diff
@@ -15,8 +15,21 @@
 
 package api
 
+import (
+	"strings"
+
+	"github.com/aws/amazon-ecs-agent/agent/config"
+	"github.com/aws/amazon-ecs-agent/agent/utils"
+	"github.com/cihub/seelog"
+	docker "github.com/fsouza/go-dockerclient"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
+)
+
 const (
 	portBindingHostIP = "0.0.0.0"
+	sepForwardSlash   = "/"
```

Comment:
Can you rename this to something like `pathSeparator`?

Reply:
You should really just use `filepath.Join` here.  If you need to manually construct for some strange reason, use `filepath.Separator`.

Reply:
@richardpen I'll remove this const
@samuelkarp thanks!